### PR TITLE
feat(updates) DP-4904 add GetUpdatedMessages API

### PR DESCRIPTION
**Pull request by dmtryyy**

### CODE REVIEW

File: proto/messaging.proto [hunk 1]
```diff
@@ -603,6 +603,7 @@ message UpdateMessageContentChanged {
     UUIDValue mid = 4 [(dlg).log="visible"]; /// Message id
     MessageContent message = 3 [(dlg).log="visible"];
     int64 edited_at = 5 [(dlg).log="visible"];
+    int64 prev_edited_at = 6;
```
Comment (nkt) [inline proto/messaging.proto]: why we need this?

File: proto/messaging.proto [hunk 2]
```diff
@@ -1056,6 +1057,23 @@ message UpdatePinnedMessagesChanged {
...
+
+message ResponseGetUpdatedMessages {
+    message UpdatedMessage {
+        UUIDValue mid = 1;
+        MessageContent message = 2;
+        int64 modified_at = 3;
+    }
+
+    repeated UpdatedMessage messages = 1;
+    int64 peer_clock = 2;
```
Comment (nkt) [inline proto/messaging.proto]: what is this? how we have to use this?

Comment (meafmira) [reply proto/messaging.proto]: What is this? Why there is no answer?

Comment (dmtryyy) [reply proto/messaging.proto]: Typical weak update, with clock of previous update to detect inconsistency

**Pull request merged**